src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -29,4 +33,26 @@
     }
     return generatedImports;
   }
+
+  public List<ImportTypeView> generateImports(SurfaceTransformerContext context) {
```

Comment:
Is this a temporary fix? I didn't need to do something like this for Java or PHP. If it is temporary, document it as such. Otherwise, if it is permanent, I am guessing it is more specific than just "imports", so I would recommend making the name more specific to what it is used for.